client/components/chart/index.jsx

```diff
@@ -60,29 +61,27 @@ module.exports = React.createClass( {
 	},
 
 	resize: function() {
-		if ( this.isMounted() ) {
```

Comment:
`isMounted` is pretty much [deprecated](https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html) and there's even an ESLint rule (`'react/no-is-mounted': 1`) about it.

In this case, it seems it was used to avoid listening to `window.resize` (which calls `this.resize` and sets the component's state) while the component is not rendered.
Though, the listener is attached when the component mounts and removed when it unmounts.
So, `this.resize` simply cannot be called outside of `componentDidMount` and `componentWillUnmount`, a period when `isMounted` will never return `false`.